app/views/v4/add-dates-errors.html

```diff
@@ -29,7 +29,7 @@
         <div class="govuk-width-container">
           <div class="govuk-form-group govuk-form-group--error">
             <label class="govuk-label" for="sort">
-              <h2 class="govuk-heading-l">Select conditions under review <i class="icon icon-important"><p class="govuk-body-s hover-hint">This information is visible to the public</p></i>
```

Comment:
If we're getting rid of the (!) alert's we should probably put something in it's place. Probably continue with the same text `This information is visible to the public` with the govuk warning text pattern

Reply:
(we can add in the beta)

Reply:
Yes. I thought about adding that, but stopped myself when I imagined it would take over quite a few fields, and some already have info / hint text alongside them. I thought it would be safest to just remove them for the moment :)